### ISSUE TITLE
Add runner: lutris-lol

### ISSUE THREAD

**Issue by mirkobrombin** [bottlesdevs/components#54]
Moved from: https://github.com/bottlesdevs/Bottles/issues/456

> Right now Bottles does not show/have the wine-lol build to run League of Legends (Lutris has one), such specific build is necessary for the game since it requires a wine hack to make the game anticheat system happy.
>
> I'm using the Flatpak version.

**Comment by mirkobrombin** [comment]
This runner cannot currently be used with the Flatpak version of Bottles due to modify_ldt being banned by flatpak

Ref: https://github.com/flathub/net.lutris.Lutris/issues/85#issue-542124343

Output example:
```bash
2021-08-12 08:12:53 INFO: Running a CMD on the wineprefix …
modify_ldt: Operation not permitted
2021-08-12 08:12:54 INFO: Running a Regedit on the wineprefix …
modify_ldt: Operation not permitted
```

**Comment by marc0x1** [comment]
This should be reopen, since https://github.com/flatpak/flatpak/pull/5082 has been merged.

**Comment by Eyap53** [comment]
It will indeed be included in the v1.15.0 (not out yet).
